src/components/earn/PoolCardTri/ClaimableRewards.tsx

```diff
@@ -0,0 +1,75 @@
+import { TokenAmount, ChainId } from '@trisolaris/sdk'
+import React from 'react'
+
+import CountUp from '../../CountUp'
+import { TYPE } from '../../../theme'
+
+import { EarnedNonTriRewards } from '../../../state/stake/stake-constants'
+import { TRI } from '../../../constants/tokens'
+import { BIG_INT_ZERO } from '../../../constants'
+
+import {
+  RewardColumn,
+  RewardsContainer,
+  StyledCurrencyLogo,
+  StyledRewardAmount,
+  StyledLongClaimableHeader,
+  StyledShortClaimableHeader,
+  StyledRewardsAmountContainer
+} from './PoolCardTri.styles'
+
+type ClaimableRewardsProps = {
+  enableClaimButton: boolean
+  noTriRewards: boolean
+  earnedAmount?: TokenAmount
+  earnedNonTriRewards?: EarnedNonTriRewards[]
+}
+
+function ClaimableRewards({
```

Comment:
I'm not able to find where this component is used, can you link it?

Reply:
Removed now from this PR as it was unused. It was there as leftover from previous design where Claimable Rewards was on collapsable section; Will add it back in expandable section PR